### DISCUSSION TITLE
Create Connection Between Component And Line Connection

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2443] [Q&A]
### Introduction

How can I create 1 connection from one component to another connection and newly created connectionstill straight?
It's like when two components connect together.

![image](https://github.com/clientIO/joint/assets/91791283/8319e3db-6c3f-4593-9e97-636889b3d539)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by QuangLanh** [comment]
Hello, please support for me !
Thank you so much.

**Comment by kumilingus** [comment]
The key is to use the [connectionPerpendicular](https://github.com/clientIO/joint/blob/master/demo/bus/src/bus.js#L22) link anchor as shown in the [mix bus](https://www.jointjs.com/demos/mix-bus) demo.

**Comment by QuangLanh** [comment]
Thank you so much! I fixed success
